src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -261,166 +394,303 @@
+    }
 
     this.business = {
-      homeJurisdiction: null, // get from COLIN
-      businessIdentifier: result.identifier,
-      businessLegalName: result.name,
-      identifier: null, // get from COLIN
-      incorporationDate: null, // get from COLIN
-      legalName: null, // get from COLIN
-      mode: 'LOOKUP',
+      bcFoundingDate: this.dateToYyyyMmDd(this.apiToDate(businessInfo.foundingDate)),
+      bcIdentifier: businessInfo.identifier,
+      bcLegalName: businessInfo.legalName,
+      homeJurisdiction: this.getHomeJurisdiction(businessInfo.jurisdiction || ''),
+      homeIdentifier: businessInfo.homeJurisdictionNumber,
+      homeIncorporationDate: this.dateToYyyyMmDd(this.apiToDate(businessInfo.homeRecognitionDate)),
+      homeLegalName: businessInfo.homeCompanyName,
+      mode: 'EXPRO',
       status: result.status,
-      taxId: result.bn?.substring(0, 9) || null
+      taxId: businessInfo.businessNumber?.substring(0, 9) || null
     }
     this.setExistingBusinessInfo(this.business)
     this.active = true
   }
 
+  /** Returns jurisdiction object for the given COLIN jurisdiction string. */
+  private getHomeJurisdiction (jurisdiction: string): any {
+    // safety check
+    if (jurisdiction) {
+      const provinces = CanJurisdictions.map(j => j.value).filter(p => p !== JurisdictionLocation.FD)
+      const states = UsaJurisdiction.map(j => j.value)
+      const countries = IntlJurisdictions.map(j => j.value)
+        .filter(c => (c !== JurisdictionLocation.CA && c !== JurisdictionLocation.US))
+
+      // look for BC, AB, etc
+      if (provinces.includes(jurisdiction)) {
+        return { country: JurisdictionLocation.CA, region: jurisdiction }
+      }
+      // look for FD
+      if (jurisdiction === JurisdictionLocation.FD) {
+        return { country: JurisdictionLocation.CA, region: 'FEDERAL' }
+      }
+      // look for "US, <state>"
+      if (
+        jurisdiction.startsWith(JurisdictionLocation.US) &&
+        states.includes(jurisdiction.substring(3))
+      ) {
+        return { country: JurisdictionLocation.US, region: jurisdiction }
+      }
+      // look for known countries
+      if (countries.includes(jurisdiction)) {
+        return { country: jurisdiction }
+      }
+    }
+    return null
+  }
+
   /** Resets this component back to its initial state. */
   reset () {
-    this.business = {} as unknown as ExistingBusinessInfoIF
+    this.business = {} as ExistingBusinessInfoIF
     this.setExistingBusinessInfo(this.business)
     // set this component to inactive (which shows the other component)
     this.active = false
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  /** Called when user has selected a jurisdiction. */
+  onJurisdictionChange (jurisdiction: any): void {
+    this.business.homeJurisdiction = null
+
+    if (jurisdiction?.group === 0) {
+      // set property reactively (in case it was null)
+      this.$set(this.business, 'homeJurisdiction', {
```

Comment:
Same thing again with `$set`.

It appears that the Vuetify v-model properties don't need this. It's likely that Vuetify just initialized and sets the properties as above.